import-export-cli/cmd/mi/delete/removeEnv.go

```diff
@@ -0,0 +1,116 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
*  Copyright (c) WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Please update [mi.go](https://github.com/wso2/product-apim-tooling/pull/978/files#diff-5d7996291df4163d59d04fc8f2436242d9e1a294095ec0c8befb0c36a9246cd0) and [addEnv.go](https://github.com/wso2/product-apim-tooling/pull/978/files#diff-5fc317f3872d32adb11ba0debb60348ef82e3fc9867b07dd877758a6bcdcdf41) as well